psi4/driver/driver_nbody.py

```diff
@@ -25,830 +25,1291 @@
 #
 # @END LICENSE
 #
-
+"""Plan, run, and assemble QC tasks to obtain many-body expansion and basis-set superposition error treatments.
+
+=============
+ManyBody Flow
+=============
+Bullet points are major actions
+Lines of dashes denote function calls
+e/d/dd=dg/g/h := energy, dipole, dipole derivative = dipole gradient, gradient, Hessian
+`mc_(frag, bas)` := a modelchem index, mc; indices of real fragments, frag; set(bas - frag) are indices of ghost fragments. see "intermediates_energy" in big table below for example.
+
+---------------------------
+ManyBodyComputer.__init__()
+---------------------------
+* not an explicit function but pydantic handles some defaults and validation
+* fields molecule, nfragments, bsse_type, return_total_data, and initial max_nbody set
+* BaseComputer.__init__()
+
+task_planner.py::task_planner()
+-------------------------------
+* computer gets modified from task_planner outside this file!
+* modelchem (method and basis) treatment levels for each n-body level determined from user levels kwarg. fields nbodies_per_mc_level set and max_nbody reset
+* for each modelchem treatment level, call build_tasks() below via one of four routes, depending on simple MB or layered MB(FD), MB(CBS), or MB(FD(CBS))
+
+    ------------------------------
+    ManyBodyComputer.build_tasks()
+    ------------------------------
+    * if supersystem requested, request (frag, bas) indices for full nbody range of nocp treatment from build_nbody_compute_list()
```

Comment:
Is it possible for `supersystem` to be _part_ of the mc level, but not _all_ of it? If so, then it's ambiguous whether this means the mc level _includes_ or _is_ the supersystem.

Reply:
no, supersystem here is the keyword sense, so it just means any n-body higher than explicitly mentioned.

Reply:
Can you change the description of `nbodies_per_mc_level` to elaborate on the significance of this option?